docs/content/token/staking/transactions.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Staking Transactions Reference
+sidebar_title: Staking Transactions Reference
+---
+
+# Overview
```

Comment:
TODO?

Reply:
Oops, yes I just forgot to write it. I'll add an overview